defaults/main.yaml

```diff
@@ -0,0 +1,18 @@
+---
+capra_lvm_filesystem_specs: []
+capra_lvm_filesystem_vg: vg
+capra_lvm_filesystem_fstype: xfs
+capra_lvm_filesystem_mount_parent: "/mnt"
+capra_lvm_filesystem_owner: "root"
+capra_lvm_filesystem_group: "root"
+capra_lvm_filesystem_mode: "0755"
+capra_lvm_filesystem_seuser: "unconfined_u"
+capra_lvm_filesystem_serole: "object_r"
+capra_lvm_filesystem_setype: "svirt_sandbox_file_t"
+capra_lvm_filesystem_selevel: "s0"
+capra_lvm_filesystem_options: "defaults"
+capra_lvm_filesystem_packages:
+  - lvm2
+  - xfsprogs
+capra_lvm_filesystem_lvcreate_options: "--wipesignatures y --zero y"
+capra_lvm_filesystem_filesystem_options: "-i size=512 -n size=8192"
```

Comment:
These _may_ be the default values built into `mkfs.xfs`. Can you verify?

Reply:
The are different. For `-I` the man page says:
> […] The minimum (and default) value is 256 bytes […]
And for `-n` the man page says:
> […] The default size value for version  2  directories  is 4096  bytes  (4  KiB),  unless the filesystem block size is larger than 4096, in which case the default value is the filesystem  block size […]

But the actual question is whether we should have them here as they are xfs specific and the role can be used for any file system.

Reply:
One could argue that the role needs to define a default filesystem and may as well define filesystem options while retaining the ability to override both. In the other case the role shouldn't install `xfsprogs` either.

Reply:
Makes sense. As the options deviate from the defaults, let us keep them.